tests/WhitespacesFixerConfigTest.php

```diff
@@ -26,7 +26,7 @@
 final class WhitespacesFixerConfigTest extends TestCase
 {
     /**
-     * @dataProvider provideTestCases
+     * @dataProvider provideCasesCases
```

Comment:
i think we can rather improve test method name 😅

Reply:
> i think we can rather improve test method name sweat_smile

Let's do it after https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7057 is merged and enabled in `@PhpCsFixer`, so it is kept synced.